packages/@icon-magic/distribute/test/index-test.ts

```diff
@@ -180,4 +193,23 @@ describe('distribute works as expected', function() {
       }
     });
   });
+
+  it('it only puts two variant into the sprite', async () => {
+    try {
+      const content = fs.readFileSync(`${output}/icons.svg`, 'utf8');
+      const doc = new DOMParser().parseFromString(content, 'svg');
+      const svgs = doc.getElementsByTagName('svg');
+      // 2 icons + parent svg
+      assert.ok(svgs.length === 3, 'Only two variants in sprite');
+      const svgIDs = ['ads-default', 'ads-default-2'];
+      svgIDs.forEach(id => {
+        assert.ok(
+          doc.getElementById(id),
+          'puts the right variants into sprite'
+        );
+      });
+    } catch (err) {
+      assert.ok(false, err);
+    }
+  });
```

Comment:
Add a test for stripping the resolution from the name too?

Reply:
it's tested when the IDs are checked in line 206.